### DISCUSSION TITLE
Wrong EPSG code for some landsat scenes?

### DISCUSSION THREAD

**Discussion by jessjaco** [microsoft/PlanetaryComputer#113] [Q&A] [answered]
I'm getting what appear to be the wrong epsg codes in items returned by a stack search

```python
import pystac_client
import planetary_computer

catalog = pystac_client.Client.open(
    "https://planetarycomputer.microsoft.com/api/stac/v1",
    modifier=planetary_computer.sign_inplace,
)

search = catalog.search(collections=["landsat-c2-l2"], datetime="2021",
    bbox=[-137.072784 , -18.57953999, -136.290558, -18.26195699])

print([item.properties['proj:epsg'] for item in search.item_collection()])

[3268, 3268, 3268, 3267, 3268, 3268, 3268, 3268, 3268, 3268, 3268, 3268, 3268, 3268, 3268, 3268, 3268, 3268, 3268, 3268, 3268, 3268, 3268, 3268]
```

Based on the location, I believe the correct zone is probably [32608](https://epsg.io/32608) (and 32607).

**Comment by TomAugspurger** [comment, marked as answer]
Thanks for the report. I believe we spotted that yesterday, and it was fixed in the stactools package at https://github.com/stactools-packages/landsat/commit/9a3d93c31f7d23c669e9c0071bb19e2d884e6a7e. According to https://github.com/stactools-packages/landsat/issues/38, it'll affect any EPSG code where the second to last digit is 0.

We need to regenerate and reingest the STAC items with the fixed EPSG codes. In the meantime, you'll have to modify the STAC items manually :/

**Reply by TomAugspurger** [reply]
These should all be fixed now. Thanks for the report. Your original example now outputs

```
[32608, 32608, 32608, 32607, 32608, 32608, 32608, 32608, 32608, 32608, 32608, 32608, 32608, 32608, 32608, 32608, 32608, 32608, 32608, 32608, 32608, 32608, 32608, 32608]
```

**Comment by jessjaco** [comment]
Thank you, will do

**Comment by jessjaco** [comment]
Found some more bad items (epsg code is 3268, should be 32608), here are the IDs: 
LC09_L2SR_045073_20220830_02_T1
LC09_L2SR_045073_20220729_02_T2
LC08_L2SR_045073_20220721_02_T2
LC09_L2SR_045073_20220627_02_T1
LC08_L2SR_045073_20220619_02_T2
LC09_L2SR_045073_20220424_02_T1

**Reply by TomAugspurger** [reply]
Hmm, thanks. I'll take a look at why those were missed.
